coordinator/src/main/scala/filodb.coordinator/NodeCoordinatorActor.scala

```diff
@@ -190,7 +191,11 @@ class NodeCoordinatorActor(metaStore: MetaStore,
       if(entries.length > 0 ) {
         entries.foreach { ingestion =>
           val data = ingestion.toString().split("\001")
-          val ref = DatasetRef(data(2), Some(data(1)))
+          val databaseOpt = if (data(1).isEmpty || data(1).equals("None")) None else Some(data(1))
```

Comment:
You don't need `data(1).equals("None")` - you are changing the serialization method to do `database.getOrElse("")` right?  When would you ever have "None"?